libs/ledger-live-common/src/families/cosmos/js-signOperation.ts

```diff
@@ -68,7 +69,7 @@ const signOperation = ({
         );
         const ledgerSigner = new LedgerSigner(transport, {
           hdPaths: [stringToPath("m/" + account.freshAddressPath)],
-          prefix: account.currency.id,
+          prefix: cryptoFactory(account.currency.id).prefix,
```

Comment:
Could you non-reg Osmosis with this change?
The value here will be different since currency.id is `osmosis` but prefix is `osmo`